### DISCUSSION TITLE
UDM-Pro SE

### DISCUSSION THREAD

**Discussion by Qlogic711** [fabianishere/udm-iptv#61] [Q&A]
I just got my UDM-SE

tried running this but getting this error

Created symlink /etc/systemd/system/multi-user.target.wants/udm-iptv.service → /lib/systemd/system/udm-iptv.service.
N: Download is performed unsandboxed as root as file '/root/igmpproxy_0.3-1_arm64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)

any thoughts?

**Comment by fabianishere** [comment]
This is a warning reported by apt. You can ignore it.

**Comment by Qlogic711** [comment]
oh ok, going to test it out today, thought something was wrong

**Comment by Qlogic711** [comment]
ok its installed but I think i am missing something

i get an IP from DHCP, but think I missing the default route

this is what I am getting

ip -4 addr show dev iptv
88: iptv@eth9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.238.128.165/22 scope global iptv
       valid_lft forever preferred_lft forever

which should be good

but under ip route list

i get this

10.11.3.153 dev ppp1 proto kernel scope link src MY-IP
10.238.128.0/22 dev iptv proto kernel scope link src 10.238.128.165
192.168.2.0/24 dev br40 proto kernel scope link src 192.168.2.1
192.168.78.0/24 dev br0 proto kernel scope link src 192.168.78.10

I think I am missing the default route

**Reply by Qlogic711** [reply]
also i noticed it says 

iptv@eth9

I am using eth10

**Reply by fabianishere** [reply]
The DHCP client explicitly asks for no default route, because the IPTV network does not route normal internet traffic.

Also, remember that Linux interfaces are zero-indexed, so WAN port 10 corresponds to eth9.

**Reply by Qlogic711** [reply]
yeah just figured it out, 

but looking at jdrom post I think I need to add the default route 

10.0.0.0/8 to 10.238.128.1

can i add that manually?

**Reply by Qlogic711** [reply]
or basically

10.2.0.0/16 to 10.238.128.1
10.125.0.0/16 to 10.238.128.1

**Reply by Qlogic711** [reply]
i just did 

ip route add 10.125.0.0/16 via 10.238.128.1
ip route add 10.2.0.0/16 via 10.238.128.1

**Reply by Qlogic711** [reply]
all working now, with the routes added.

Thank You
